kube/services/revproxy/gen3.nginx.conf/README.md

```diff
@@ -0,0 +1,3 @@
+# TL;DR
+
+Decomposed nginx reverse proxy configuration.  Use `gen3 kube-setup-revproxy` to update the revproxy config.
```

Comment:
can you add some brief explaination about how it works, eg: it's based on checking whether the k8s service is up for each api

Reply:
Good idea.  Done!